### PR TITLE
Store logs from WASM execution in the state using Receipt's nonce as a key

### PR DESCRIPTION
**Pull request by evgenykuzyakov**
Have to change WASM external API from "log" to "debug", otherwise rust wasm fails to compile with signature conflict.
Added test to test logs storage and propagation.

### CODE REVIEW

**Comment by evgenykuzyakov** [comment]
Fixes #361

**Comment by evgenykuzyakov** [comment]
Ah. Need to update tests, since I'm changing API from "log" to "debug"

**Comment by evgenykuzyakov** [comment]
Even better solution. I'll just introduce alias to "log" to "debug" instead of replacing it.

File: node/runtime/src/lib.rs [hunk 1]
```diff
@@ -78,6 +79,14 @@ fn account_id_to_bytes(col: &[u8], account_key: &AccountId) -> Vec<u8> {
     key
 }
 
+// TODO(#362): Maybe need to replace Receipt nonce with the Receipt hash.
+// It's actually might be fine as well, since nonce is unique.
```
Comment (bowenwang1996) [inline node/runtime/src/lib.rs]: I think it's fine. Should we remove the issue then?

Comment (evgenykuzyakov) [reply node/runtime/src/lib.rs]: Depends on the future implementation of block explorer

**Pull request merged**